.appveyor.yml

```diff
@@ -67,7 +68,7 @@ for:
 
 cache:
   - $(RUST_DIR)
-  - $(APPVEYOR_BUILD_FOLDER)\target\release\
+  - $(CARGO_HOME)
```

Comment:
This directory is already getting cached, because it's under $RUST_DIR. 
Let's not do it twice.

Reply:
Good catch. Fixed.

Reply:
In fact, it might potentially be necessary to add something similar to the `- rm -rf "$TRAVIS_HOME/.cargo/registry/src` command on travis at some point.
Not that I think that it's all that important ATM.